### PR TITLE
Detect bite opportunities in forked urns, update Vulcanize queries

### PR DESCRIPTION
**Pull request by EdNoepel**
**Changes**
-  When finding `bite` opportunities using chain data, the current implementation misses urns which were `fork`ed from other urns, yet never `frob`bed.  This change includes a `pymaker` update allowing the keeper to scrape logs for `fork` and `move` events, uncovering these urns.
- Vulcanize query grammar has changed.  This update supports the new `getUrnsByIlk` query, request limits, and new health metrics.
- Improvements to the urn cache reconciliation manual test have been made.
- The recommended MANA-A `--from-block` for mainnet was corrected.

### CODE REVIEW

File: auction_keeper/urn_history.py [hunk 1]
```diff
@@ -57,17 +57,25 @@ def get_urns(self) -> Dict[Address, Urn]:
...
-                                        chunk_size=self.chunk_size)
-        for frob in frobs:
-            urn_addresses.add(frob.urn)
+        logs = self.mcd.vat.past_logs(from_block=from_block, to_block=to_block, ilk=self.ilk,
+                                      chunk_size=self.chunk_size)
+        for log in logs:
+            if isinstance(log, Vat.LogFrob):
+                urn_addresses.add(log.urn)
+            if isinstance(log, Vat.LogFork) or isinstance(log, Vat.LogMove):
+                urn_addresses.add(log.src)
```
Comment (KentonPrescott) [inline auction_keeper/urn_history.py:66]: A `src` that is forking some of its debt to `dst` would have needed to previously `frobbed`, so recording `log.src` may not be needed, as `src` would have been captured in a `LogFrob`.

**Review by KentonPrescott — APPROVED** [review]
LGTM

File: auction_keeper/urn_history.py (hunk 1 above)
Comment (EdNoepel) [reply auction_keeper/urn_history.py:66]: I considered the case the urn A was forked to urn B which was forked to urn C.  In that case, urn B would never have been frobbed, but it also would've been the `dst` of the first fork, so it'd be captured anyway.  As such; I agree.  I'll remove line 66 and retest.

Comment (EdNoepel) [reply auction_keeper/urn_history.py]: Retested; no issues with the change.

**Pull request merged**